packages/e2e-tests/test-applications/standard-frontend-react/src/index.tsx

```diff
@@ -15,7 +15,8 @@ import Index from './pages/Index';
 import User from './pages/User';
 
 Sentry.init({
-  dsn: 'https://public@dsn.ingest.sentry.io/1337',
+  // DSN belongs to "e2e-javascript-standard-frontend-react" project in "sentry-sdks" org
+  dsn: 'https://db311937c30c4c96bf06f4d1f08b2ad1@o447951.ingest.sentry.io/4503941750587392',
```

Comment:
l: Should we inject this instead of hardcoding?

m: Also, I'd rather just send everything to one project (not specify it to react), we can use sdk name/version to filter it in the Sentry UI.

Reply:
Thinking more about it I agree. Changed it so we're injecting the DSN via env var: 0f35fe19d7d992af064555800c579c09f20eba76